pkg/apis/garden/validation/validation_test.go

```diff
@@ -2800,6 +2804,18 @@ var _ = Describe("validation", func() {
 				}))
 			})
 
+			It("should forbid very frequent backup configuration", func() {
+				shoot.Spec.Backup = invalidFrequentBackup
+
+				errorList := ValidateShoot(shoot)
+
+				Expect(len(errorList)).To(Equal(1))
+				Expect(*errorList[0]).To(MatchFields(IgnoreExtras, Fields{
```

Comment:
Please use `ConsistsOf`

Reply:
Done, we should later think about "migrating" all `MatchFields` to `ConsistsOf` to avoid in-consistency.

Reply:
Sure, no worries, feel free to file a separate PR that does that :)